lms/djangoapps/certificates/signals.py

```diff
@@ -76,10 +78,10 @@ def listen_for_passing_grade(sender, user, course_id, **kwargs):  # pylint: disa
     Listen for a learner passing a course, send cert generation task,
     downstream signal from COURSE_GRADE_CHANGED
     """
-    if is_using_certificate_allowlist_and_is_on_allowlist(user, course_id):
-        log.info(f'{course_id} is using allowlist certificates, and the user {user.id} is on its allowlist. Attempt '
-                 f'will be made to generate an allowlist certificate as a passing grade was received.')
-        return generate_allowlist_certificate_task(user, course_id)
+    if can_generate_certificate_task(user, course_id):
+        log.info(f'{course_id} is using V2 certificates. Attempt will be made to generate a V2 certificate for '
+                 f'{user.id} as a passing grade was received.')
+        return generate_certificate_task(user, course_id)
 
     if not auto_certificate_generation_enabled():
```

Comment:
Random thought. Should this check happen before the can_generate_certificate_task? Maybe we should hold off to making any changes until we look into auto-certificate-generation some more in that other subtask?

Reply:
I agree with both thoughts. :-) 

1) yes we'll likely move this check and 2) it'll happen in a subsequent PR